vyxal/elements.py

```diff
@@ -4009,12 +4009,18 @@ def vy_bin(lhs, ctx):
     (str) -> binary of each codepoint
     """
     ts = vy_type(lhs)
-    return {
-        (NUMBER_TYPE): lambda: [int(x) for x in bin(int(lhs))[2:]],
-        (str): lambda: vectorise(
-            vy_bin, wrapify(chr_ord(lhs, ctx=ctx), None, ctx), ctx=ctx
-        ),
-    }.get(ts, lambda: vectorise(vy_bin, lhs, ctx=ctx))()
+    if ts == (NUMBER_TYPE):
```

Comment:
why the parens here? Is it supposed to be a tuple?

Reply:
Looks like in [vy-type](https://github.com/Vyxal/Vyxal/blob/d82539759c2379042bf987c5fa0e2d975499d239/vyxal/elements.py#L4367) only `NUMBER_TYPE` without parentheses are returned.

Reply:
It works and passes the tests and everything, even with the parentheses, but I think I'll remove them anyway because I can and because why not

Reply:
`2 == (2)` returns `True` in python.

Reply:
That's probably typical python then